README.md

```diff
@@ -118,11 +116,13 @@ Contour is an Ingress controller for Kubernetes that works by deploying the Envo
 
 ### Contour Versions
 
-- Download latest version: [Contour v1.18.2](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=contour/v1.18.2/contour.yml)
+- Download latest version: [Contour v1.24.4](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=contour/v1.24.4/contour.yml)
+- Download version: [Contour v1.18.2](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=contour/v1.18.2/contour.yml)
 
 Contour Sample `values.yaml`
 
-- Download the latest version: [values for v1.18.2](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=contour/v1.18.2/contour-data-values.yml). These values can be used _as-is_ and require no configuration changes.
+- Download latest version: [values for v1.24.4](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=contour/v1.24.4/contour-data-values.yml). These values can be used _as-is_ and require no configuration changes.
```

Comment:
Are the default data-values files identical for both versions? If so maybe we can skip versioning these for now and just link one.

Reply:
Yes, they are the same data values. Do you also want to move contour/v1.18.2/contour-data-values.yml up one level to contour/contour-data-values.yml path?

Reply:
Are you referring to the path on jfrog? It might not matter from this repo, we could just point at the latest.

Reply:
Yeah, I meant the path on jfrog.